src/arc.ts

```diff
@@ -154,6 +154,7 @@ export async function initializeArc(provider?: Web3Provider): Promise<boolean> {
     } else {
       arc = new Arc(arcSettings);
     }
+    arc.web3.pollingInterval = 60000;
```

Comment:
this already been set https://github.com/daostack/alchemy/blob/dev-2/src/arc.ts#L157